content/en/news/releases/1.21.x/announcing-1.21/change-notes/index.md

```diff
@@ -0,0 +1,241 @@
+---
```

Comment:
I think for this minor release, the `minorReleaseNotes.md` should be used, which has sorted notes by groups. In patch releases, we use `releaseNotes.md`.

Reply:
Yes, minor release notes have things broken up by category whereas releaseNotes are for patch releases which don't.